extensions-contrib/distinctcount/src/main/java/org/apache/druid/query/aggregation/distinctcount/DistinctCountAggregatorFactory.java

```diff
@@ -116,15 +116,6 @@ public int compare(Object o, Object o1)
   @Override
   public Object combine(Object lhs, Object rhs)
   {
-    if (lhs == null && rhs == null) {
-      return 0L;
-    }
-    if (rhs == null) {
-      return ((Number) lhs).longValue();
-    }
-    if (lhs == null) {
-      return ((Number) rhs).longValue();
-    }
     return ((Number) lhs).longValue() + ((Number) rhs).longValue();
```

Comment:
This change makes `combine` no longer work on nulls; was that not needed for some reason?